src/Http/Requests/Bulk/DestroyThreads.php

```diff
@@ -33,29 +33,44 @@ public function authorizeValidated(): bool
 
     public function fulfill()
     {
-        $threads = $this->threads();
+        $threads = $this->threads()->get();
+
+        if ($threads->count() === 0) return 0;
+        
+        // Avoid using Eloquent to prevent automatic touching of updated_at
+        $query = \DB::table((new Thread)->getTable())
```

Comment:
You should import the `DB` facade, i myself (and some others) have disabled laravel's class aliasses and therefore this wouldn't work.